src/components/visualization/bar-chart/CustomTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import { DefaultTooltipContent, type TooltipProps } from 'recharts';
+import { computeIfAbsent } from '@/utils/mapUtils';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+interface CustomTooltipProps extends TooltipProps<any, any> {
```

Comment:
If you change `any` to `unknown` you probably won't need the eslint-disable

Reply:
Unfortunately unknown doesn't fulfill the constraints that TooltipProps excepts to be fulfilled here. Avoiding any would probably require some quite complicated typing here